tests/test_api.py

```diff
@@ -196,6 +196,18 @@ def test_do_GET(self):
             self.assertIsNotNone(MockRestApiServer(RestApiHandler, 'GET /patroni'))
         with patch.object(MockHa, 'is_standby_cluster', Mock(return_value=True)):
             MockRestApiServer(RestApiHandler, 'GET /standby_leader')
+        with patch.object(RestApiHandler, 'get_postgresql_status', Mock(return_value={
+            'state': 'running', 'postmaster_start_time': '2021-02-20 02:07:35.662 UTC',
+            'role': 'master', 'server_version': '100015', 'cluster_unlocked': 'false',
+            'timeline': '14', 'xlog': {'location': '50337208'}}
+        )):
+            MockRestApiServer(RestApiHandler, 'GET /metrics')
+        with patch.object(RestApiHandler, 'get_postgresql_status', Mock(return_value={
+            'state': 'running', 'postmaster_start_time': '2021-02-20 02:07:35.662 UTC',
+            'role': 'replica', 'server_version': '100015', 'cluster_unlocked': 'false',
+            'timeline': '14', 'xlog': {'received_location': '50337208'}}
+        )):
+            MockRestApiServer(RestApiHandler, 'GET /metrics')
```

Comment:
Not required, because https://github.com/zalando/patroni/pull/1848/files#diff-3451cc474d5698222168f66a293339a6bd296ee2c56f56409866bbae99917a79R251-R254 does everything we need.
```suggestion
```